cves/CVE-2013-4316.yml

```diff
@@ -60,13 +66,13 @@ fixes_vcc_instructions: |
   Please put the SVN commit number in "commit" below, and any notes about how this
   was discovered in the "note" field.
 fixes:
-  - commit: 58947c3f85ae641c1a476316a2888e53605948d1
+  - commit: c643336945dda84cbcdc8a39530baa24fede28c4
     note: Disables allowing dynamic method calls by default
```

Comment:
If this feature was enabled, was the vulnerability still there?

Reply:
Added a quick note to say that users could still enable this if they wanted